src/gemm_v2.cpp

```diff
@@ -70,6 +70,8 @@
 /// Let's pass literal value as a workaround; there should be no harm.
 #define USE_GEMM_FLAGS_FP16_ALT_IMPL_242 (MIOPEN_ROCBLAS_VERSION_FLAT == 2042000)
 
+#define F8_SUPPORTED (MIOPEN_ROCBLAS_VERSION_FLAT > 3000000)
```

Comment:
@atamazov  Why not reuse the existing macro `USE_ROCBLAS_GEMM_EX3` ?

Reply:
@JehandadKhan I've missed that is has the same semantics. Let' me generalize the code.

Reply:
@JehandadKhan [resolved]